src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Listeners/EventHubListener.cs

```diff
@@ -106,8 +106,20 @@ public Task OpenAsync(PartitionContext context)
 
             public Task ProcessErrorAsync(PartitionContext context, Exception error)
             {
-                string errorMessage = $"Error processing event from Partition Id:{context.PartitionId}, Owner:{context.Owner}, EventHubPath:{context.EventHubPath}";
-                _logger.LogError(error, errorMessage);
+                string errorDetails = $"Partition Id:{context.PartitionId}, Owner:{context.Owner}, EventHubPath:{context.EventHubPath}";
+
+                if (error is ReceiverDisconnectedException ||
+                    error is LeaseLostException)
+                {
+                    // For EventProcessorHost these exceptions can happen as part
+                    // of normal partition balancing across instances, so we want to
+                    // trace them, but not treat them as errors.
+                    _logger.LogInformation($"An Event Hub exception of type '{error.GetType().Name}' was thrown from {errorDetails}. This exception type is typically a result of Event Hub processor rebalancing and can be safely ignored.");
```

Comment:
@jeffhollan -- any suggestions for this message?

Reply:
This looks good to me. Thanks